setup.py

```diff
@@ -74,7 +74,7 @@
         "Topic :: System :: Monitoring",
         ],
     install_requires=[
-        'requests<=2.10.0',
+        'requests>=0.12.1,<=2.10.0',
```

Comment:
@coryvirok But wouldn't `<=` mean that if you are using pyrollbar, you can't upgrade requests (for the purposes of the rest of your application) to a version beyond that? That seems worse to me.